README.md

```diff
@@ -97,7 +98,53 @@ sudo apt install libwebkit2gtk-4.1-dev
 sudo dnf install gtk3-devel webkit2gtk4.1-devel
 ```
 
-Fedora does not have the Ayatana package yet, so you need to use the GTK one, see the [feature flags documentation](https://docs.rs/wry/latest/wry/#feature-flags).
+> [!WARNING]
+> Fedora does not have the Ayatana package yet, so you need to use the GTK one, see the [feature flags documentation](https://docs.rs/wry/latest/wry/#feature-flags).
+
+### Nix & NixOS
+
+ ```Nix
+ let
+    # Unstable Channel | Rolling Release
+    pkgs = import (fetchTarball("channel:nixpkgs-unstable")) { };
+    packages = with pkgs; [
+      pkg-config
+      webkitgtk_4_1
+      libayatana-appindicator
+      libappindicator-gtk3
```

Comment:
these two packages are no longer a concern for this repo, we should remove them from here